README.md

```diff
@@ -16,7 +16,7 @@ returned object of `new WebSocket()` calls with the socket you passed in.
 
 ```javascript
 const socket = new Fauxnix(function() {
-  this.receive("topic", "event", function() {
+  this.receive("topic", "event", function(payload) {
     return { status: "ok", response: { id: 1 } };
```

Comment:
It's more of an example than anything.

Would `response: { id: payload.id }` work?